packages/xo-web/src/common/editable/index.js

```diff
@@ -150,6 +150,17 @@ class Editable extends Component {
 
   render() {
     const { state, props } = this
+    const { error, saving } = state
+
+    const ErrorToolTip = props =>
```

Comment:
Let's keep it homogeneous and use the casing we used everywhere so far: `ErrorTooltip`.